_posts/2020-10-01-annual-report-part-2.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: OLS Project and Community report - Part 2
+authors:
+- bebatut
+- malvikasharan
+- yochannah
+image: https://images.unsplash.com/photo-1523540939399-141cbff6a8d7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80
+photos:
+  name: Sam Dan Truong, Unsplash id: @sam_truong
+  license: CC-BY
+  url: https://unsplash.com
+---
+
+# Year one of Open Life Science (OLS) - Part 2
+
+*This is the second part of the 3 parts OLS project and community report: **How far we have come***
+
+## OLS-1 to OLS-2: *cohort doubled in size*
```

Comment:
In the previous part, we discussed OLS-1. But at this point, I am happy to actually merge all these broken parts into one file as suggested by you so that this can go under the OLS-1 section.